snippets/article-card.liquid

```diff
@@ -10,6 +10,7 @@
     - show_date: {String} The setting either show the article date or not. If it's not included it will not show the image by default
     - show_author: {String} The setting either show the article author or not. If it's not included it will not show the author by default
     - show_badge: {String} The setting either show the blog badge or not.
+    - lazy_load: {Boolean} Image should be lazy loaded. Default: true (optional)
```

Comment:
Since article card is called in `featured-blogs` and `main-blog` as well, do they we need to pass that variable there ? I think we do 🤔